app/views/groups/index.html.erb

```diff
@@ -1,11 +1,18 @@
-<section class="section-1">
+<%= render partial: 'shared/nav', locals: {title:'CATEGORIES', symbol:'', url: ''}%>
+
+<section class="body_container">
   <p> <%= notice %></p>
-  <p> Total expenses for <%= current_user.name.titleize %> : $<%=@total_expenses %> </p>
-  
+  <div  class="total_box"> 
+  <p class="group_name"> Total expenses for <%= current_user.name.titleize %> </p>
+   <p class="total_amount"> $<%=@total_expenses %> </p>
+  </div>
   <div>
-  <% @groups.each do |group| %>
+  <% @groups.includes(:expenses).each do |group| %>
   <%= render partial: 'shared/group', locals: { group: group } %>
 <%end%>
       </div>
-      <%= button_to 'ADD CATEGORY', new_user_group_path(user_id: current_user.id), method: :get, class: ''%>
+      <div class="button_box">
+      <div class="push_box">
+      <%= button_to 'ADD CATEGORY', new_user_group_path(user_id: current_user.id), method: :get, class: 'add_category fixed-bottom'%> </div>
+      </div>
```

Comment:
- [ ] Kindly improve the design, Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [ ] Kindly follow the provided design (width, font, size, height, background, color...)
1. Make sure to use the same background color on each category container. 👍🏻 
3. Remove the green section on the middle of your page that appeared the first time the user visits your page. 👍🏻 
3. Add this white border around the Add New Category button text. 👍🏻 
4. Make sure you respect the dimension of the button (width, height, font size, padding)...
5. The `Add Transaction` button has a similar problem. Kindly fix it as well. 💯 👍🏻 

Your Implementation | Original Design
:---:|:---:
![image](https://github.com/manq2010/budget-app/assets/79058364/47cc8c91-e5ff-4e56-8528-3a81f4030c69) | ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)